index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+    <title>web page portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav class="container">
+            <h2 class="baritem">Welcome</h2>
+            <img src="./Union.svg" alt="menu">
+        </nav>
```

Comment:
As the l[ast code reviewer suggested ](https://github.com/PierreBarba/portfolio-setup-and-movile-first/pull/1#discussion_r1007460620)please remove the `nav` tag and wrap just the hamburger menu with a `nav` tag. This is for semantic reasons and later in future projects, you will use this button to create a functional hamburger menu. :+1: :smiley: :smiley: